components/director/internal/open_resource_discovery/client.go

```diff
@@ -115,6 +127,11 @@ func (c *client) FetchOpenResourceDiscoveryDocuments(ctx context.Context, app *m
 				return
 			}
 
+			if docDetails.Perspective == SystemVersionPerspective {
```

Comment:
Should we check if` docDetails.Perspective` is empty before getting the value?

Reply:
No, we've agreed that if the perspective is not explicitly defined, then we will fallback to SystemInstancePerspective